scraper/scraper.py

```diff
@@ -0,0 +1,209 @@
+        "gmtOff": gmtOff
+    }
+    
+    if completed:
+        resultLink = containerDiv.find('a')['href']
+        result = getEventResults(resultLink)
+        event['results'] = result
+        
+    return event
+
+
+def getCircuit(url):
+    print("Getting Circuit details...")
+    html = requests.get(url)
+    soup = BeautifulSoup(html.text.encode('utf-8'), 'html.parser')
+    
+    flag = soup.find('span', class_='f1-flag--wrapper')
+    trackName = flag.next_sibling.next_sibling.string
+    
+    stats = soup.find_all('div', class_='f1-stat')
+    
+    circuit = {"trackName": trackName}
+    for stat in stats:
+        contents = [content for content in stat.stripped_strings]
+        circuit[camelCase(contents[0])] = " ".join(contents[1:])
+    
+    return circuit
+    
+    
+def isTrackMap(alt):
+    return alt and 'carbon.png' in alt
+
+def getRace(url, round_):
+    html = requests.get(url)
+
+    soup = BeautifulSoup(html.text.encode('utf-8'), 'html.parser')
+
+    raceLocation = soup.find('h1', class_='race-location').contents[0]
+    raceName = soup.find('h2', class_='f1--s').string
+    trackMapImg = soup.find('img', alt=isTrackMap)['data-src']
+
+    upcoming = soup.find_all('div', class_='upcoming')
+    completed = soup.find_all('div', class_='completed')
+
+    events = []
+    for eventDiv in upcoming:
+        events.append(getEventDetails(eventDiv))
+        
+    for eventDiv in completed:
+        events.append(getEventDetails(eventDiv, completed=True))
+    
+    splitUrl = url.split(".")
+    splitUrl.pop()
+    circuit = getCircuit(".".join(splitUrl) + "/Circuit.html")
+    
+    
+    return{
+            "round": round_,
+            "raceName" : raceName, 
+            "raceLocation": raceLocation, 
+            "events": events[::-1], 
+            "trackMap": trackMapImg,
+            "circuit": circuit
+        }
+
+
+def getStanding(url):
+    html = requests.get(url)
+    soup = BeautifulSoup(html.text.encode('utf-8'), 'html.parser')
+    
+    standingsTable = soup.find('table', class_='resultsarchive-table')
+    
+    infos = [info.lower() for info in standingsTable.thead.stripped_strings]
+    
+    standings = []
+    for child in standingsTable.tbody.children:
+        if child != "\n":
+            data = []
+            for ch in child.children:
+                joined = " ".join(ch.stripped_strings)
+                if joined:
+                    data.append(joined)
+                
+            result = {}
+            for ind, res in enumerate(data):
+                result[infos[ind]] = res
+                
+            standings.append(result)
+            
+    return standings
+
+
+def getStandings(year):
+    driversUrl = f"https://www.formula1.com/en/results.html/{str(year)}/drivers.html"
+    teamsUrl = f"https://www.formula1.com/en/results.html/{str(year)}/team.html"
+
+    return {"drivers": getStanding(driversUrl), "teams": getStanding(teamsUrl)}
+
+
+def motherShip(year):
```

Comment:
The motherShip function could be broken down into smaller functions to improve readability and maintainability.